tdesign/src/Menu/Menu.tsx

```diff
@@ -53,7 +53,7 @@ const Menu = ({ children, style = {} }: MenuProps) => {
         expanded ? "initial" : "surfaces.background.main",
         "initial",
       ],
-      padding: "1em",
+      padding: "1em 0.5em",
```

Comment:
As discussed, let's avoid this shorthand.

Not worth fixing on this branch though.

Also, note that this Menu component is used in more than just the repository page (I think the other place is Settings)